.gitignore

```diff
@@ -1 +1,4 @@
-node_modules
\ No newline at end of file
+node_modules
+coverage
+.nyc_output
+coverage.lcov
```

Comment:
No need to ignore `coverage.lcov`, if you're piping output from `nyc` to `coveralls`, like I mentioned in the comment below.

Reply:
Ok so you mean i should push all of those dir/files with all my local reports & meta data i used for my local test bed? Should not they be ignored like `npm rebuild` as they will be created always while codecov report generation and uploading via nyc process will be running? Please suggest.

Reply:
I was talking only about `coverage.lcov`. But since I was thinking about coveralls and piping output before, it can stay there.